loader.cpp

```diff
@@ -145,7 +145,12 @@ bool wibo::Executable::loadPE(FILE *file) {
 
 	// Build buffer
 	imageSize = header32.sizeOfImage;
-	imageBuffer = mmap((void *) header32.imageBase, header32.sizeOfImage, PROT_READ|PROT_WRITE|PROT_EXEC, MAP_ANONYMOUS|MAP_FIXED|MAP_PRIVATE, -1, 0);
+	if (exec) {
+		imageBuffer = mmap((void *)header32.imageBase, header32.sizeOfImage, PROT_READ | PROT_WRITE | PROT_EXEC,
+						   MAP_ANONYMOUS | MAP_FIXED | MAP_PRIVATE, -1, 0);
+	} else {
+		imageBuffer = malloc(header32.sizeOfImage);
```

Comment:
should we use mmap here too, for consistency? I could imagine alignment mattering